templates/security/cve/filters/releases-filter.html

```diff
@@ -0,0 +1,32 @@
+<form>
+  <label class="p-checkbox">
+    <input type="checkbox" aria-labelledby="allMaintainedReleases" class="p-checkbox__input" />
+    <span class="p-checkbox__label" id="allMaintainedReleases">All maintained releases</span>
+  </label>
+  <label class="p-checkbox">
+    <input type="checkbox" aria-labelledby="22.10" class="p-checkbox__input" />
+    <span class="p-checkbox__label" id="22.10">22.10 kinetic</span>
```

Comment:
Is this list complete, it seems like there are some more recent versions missing? ie. 23.04, 23.10

Reply:
It is yeah, I added the link to the figma file just now and that's what it's supposed to match